tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,22 +6,7 @@
    "source": [
```

Comment:
<p>I think we should change <code>circ</code> to <code>qc</code> or <code>circuit</code> . I remember there were some discussions about this but I can't remember what was the conclusion. @1ucian0 do you have preference?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>